pylint/checkers/exceptions.py

```diff
@@ -168,7 +173,11 @@ def _is_raising(body: list[nodes.NodeNG]) -> bool:
     "W0719": (
         "Raising too general exception: %s",
         "broad-exception-raised",
-        "Used when an except raises a too general exception.",
+        "Raising exceptions that are too generic force you to catch exceptions "
+        "generically too. It will force you to use a naked ``except Exception:``"
+        " clause. You might then end up catching exceptions other than the ones "
+        "you expect to catch. This can hide bugs or make it harder to debug programs"
+        " when unrelated errors are hidden.",
```

Comment:
```suggestion
        "Raising exceptions that are too generic force you to catch exceptions "
        "generically too. It will force you to use a naked ``except Exception:`` "
        "clause. You might then end up catching exceptions other than the ones "
        "you expect to catch. This can hide bugs or make it harder to debug programs "
        "when unrelated errors are hidden. =",
```

Reply:
Can't apply suggestion because there's a '=' at the end. Let's merge, we have bigger issues to fix :)

Reply:
I'll edit